packages/framework/fluid-static/api-report/fluid-static.api.md

```diff
@@ -40,7 +40,7 @@ export class DOProviderContainerRuntimeFactory extends BaseContainerRuntimeFacto
 // @public
 export class FluidContainer<TContainerSchema extends ContainerSchema = ContainerSchema> extends TypedEventEmitter<IFluidContainerEvents> implements IFluidContainer<TContainerSchema> {
     constructor(container: IContainer, rootDataObject: IRootDataObject);
-    attach(): Promise<string>;
+    attach(filePath?: string, fileName?: string): Promise<string>;
```

Comment:
i don't think this is the correct way to expose this. i would remove it from this pr, and do a separate PR that only does this. the problem you will have to solve is that this type is also used by the azure client, but it does not take these parametere

Reply:
also, exposing as an object, rather than splayed params will likely be a better path, as it is more supportable and maintainable

Reply:
another change we should make, but no in this PR, is to make the return type of attach an object, not a string, this will allow us to return named things, like {itemId}, {containerId} in a way this is also extendable such that we could expose more data here if we have use, the raw string it too restrictive.